src/asf_tools/water_map.py

```diff
@@ -390,6 +390,15 @@ def hyp3():
 
     log.info(f'Water map created successfully: {water_map_raster}')
 
+    files_to_remove = [
+        Path(product_name) / f'{product_name}_VV_initial.tif',
+        Path(product_name) / f'{product_name}_VH_initial.tif',
+        Path(product_name) / f'{product_name}_VV_fuzzy.tif',
+        Path(product_name) / f'{product_name}_VH_fuzzy.tif',
```

Comment:
I'm just being nit-picky, but may as well use `product_dir` since it was just defined above.
```suggestion
        product_dir / f'{product_name}_VV_initial.tif',
        product_dir / f'{product_name}_VH_initial.tif',
        product_dir / f'{product_name}_VV_fuzzy.tif',
        product_dir / f'{product_name}_VH_fuzzy.tif',
```